clib/mininet_test_base.py

```diff
@@ -223,6 +223,16 @@ def _set_vars(self):
     def _set_log_level(self, name='faucet'):
         self._set_var(name, 'FAUCET_LOG_LEVEL', str(self.LOG_LEVEL))
 
+    def _set_event_log(self, timeout):
+      event_log = os.path.join(self.tmpdir, 'event.log')
+      controller = self._get_controller()
+      sock = self.env['faucet']['FAUCET_EVENT_SOCK']
```

Comment:
I'm seeing this same bit of code used ~3 places in the existing code (before your PR) -- can you be a good code citizen and clean those up?  E.g., make them call this function rather than doing the FAUCET_EVENT_SOCK thing directly?